.travis.yml

```diff
@@ -1,13 +1,14 @@
 script:
   - mkdir build
   - cd build
-  - cmake ${CMAKEFLAGS} -DCMAKE_CXX_COMPILER=${CXX} -DCMAKE_C_COMPILER=${CC} ..
+  - cmake ${CMAKEFLAGS} -DCMAKE_CXX_COMPILER=${CXX} ..
   - make
   - make test
   - make package
 
 sudo: required
 dist: trusty
+language: cpp
```

Comment:
Looks like it doesn't (it picked up g++ 4.8 and clang++-3.5). I can revert those two lines.

For one of my other projects, I remember assigning the compiler name to another variable and using that to set CXX in the install phase (it runs before script). Let me know if you want me to make such a change instead.